tests/unit/lms/resources/_js_config_test.py

```diff
@@ -187,31 +188,24 @@ def test_it_raises_if_theres_no_ApplicationInstance(
 class TestAddCanvasFileID:
     """Unit tests for JSConfig.add_canvas_file_id()."""
 
-    def test_it_adds_the_viaUrl_api_config(self, js_config):
-        js_config.add_canvas_file_id(
-            "example_canvas_course_id", "example_canvas_file_id"
```

Comment:
I don't think we need to prefix strings with "example_" or "test_" in tests, it's kind of obvious by context.